pom.xml

```
@@ -5,14 +5,14 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.flowingcode.vaadin.addons</groupId>
-    <artifactId>template-addon</artifactId>
+    <artifactId>date-time-range-picker-addon</artifactId>
     <version>1.0.0-SNAPSHOT</version>
-    <name>Template Add-on</name>
-    <description>Template Add-on for Vaadin Flow</description>
+    <name>DateTimeRangePicker</name>
+    <description>DateTimeRangePicker Vaadin Add-on</description>
```

"Vaadin" [is a registered trademark](https://vaadin.com/trademark). You may name you application "XYZ for Vaadin", but you may not name it "Vaadin XYZ"

```diff
-    <description>DateTimeRangePicker Vaadin Add-on</description>
+    <description>DateTimeRangePicker component for Vaadin</description>
```

but, insteed of that, prefer focusing on the component's function and benefit to the user, rather than just its name and platform association (FlowingCode/TechnicalDocs#93):
```suggestion
    <description>Component that facilitates the selection of a date and time range</description>
```